src/main/java/io/github/cottonmc/cotton/gui/SyncedGuiDescription.java

```diff
@@ -148,18 +148,20 @@ public void onSlotClick(int slotNumber, int button, SlotActionType action, Playe
 						return;
 					}
 				}
-				
+
 				if (toTransfer.isEmpty()) {
 					slot.setStack(ItemStack.EMPTY);
 				} else {
 					slot.markDirty();
 				}
 			}
-			
+
+		} else {
+
 			super.onSlotClick(slotNumber, button, action, player);
 		}
```

Comment:
Instead of adding the `else`, you should just move the `super.onSlotClick` call to be after the if block (I think this is actually my fault from the initial 1.17 port)

Reply:
I added the else as this is what it was in the last 3.x release. I thought this would be correct to re-add because otherwise ScreenHandler#internalOnSlotClick's QUICKMOVE action will also be fired. I'll change this though if you think this is correct :) And indentation changes will be fixed once I receive your answer :)

Reply:
Yeah, the else actually seems to be correct - so my fault when porting as I guessed :^)

That should be fine if you also fix the spacing around there (remove the two empty lines around `} else {`).

Reply:
Sounds good; I shall push these changes now :)